### PR TITLE
Convert to JPEG on upload when png file size is large

### PR DESCRIPTION
**Pull request by pastak**

### CODE REVIEW

File: src/libs/convertAdjustmentJpegQuality.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export default (canvas) => {
+  let quality = QUALITY_MAX
+  let result = canvas.toDataURL('image/jpeg')
+  if (result.length < LIMIT_VOLUME) return result
+  quality -= (QUALITY_MAX - QUALITY_MIN) / 2
+
+  for (let i = 0; i < 5; i++) {
+    result = canvas.toDataURL('image/jpeg', quality)
+    if (result.length <= LIMIT_VOLUME) {
+      quality += (QUALITY_MAX = QUALITY_MIN) / Math.pow(i + 2)
```
Comment (rakusai) [inline src/libs/convertAdjustmentJpegQuality.js]: Is this possible?  `QUALITY_MAX = QUALITY_MIN`

Comment (pastak) [inline src/libs/convertAdjustmentJpegQuality.js]: typo `-`

File: src/constants/index.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 export const ESC_KEY_CODE = 27
 export const JACKUP_MARGIN = 30
+export const UPLOAD_LIMIT_VOLUME = 2 * 1024 * 1024 * ( 4 / 3 ) // = 2 MB
```
Comment (rakusai) [inline src/constants/index.js]: needs comment for 4 / 3
```
// = 2 MB when Base64 is decoded
```

**Comment by rakusai** [comment]
LGTM.
Please fix the test.

**Pull request merged**